requirements.txt

```diff
@@ -29,3 +29,4 @@ uWSGI==2.0.5
 wsgiref==0.1.2
 smarturls==0.1.3
 Fabric==1.9.0
+psycopg2
```

Comment:
Postgresql is used just in production, in dev normally we use sqlite, even the `settings.py` sets a sqlite db.
This may broke future instalations of the speakerfight, because this package needs that the users have the pgsql client.